contracts/0.8.25/vaults/PinnedBeaconProxy.sol

```diff
@@ -36,4 +36,8 @@ contract PinnedBeaconProxy is BeaconProxy {
     function implementation() external view returns (address) {
         return _implementation();
     }
+
```

Comment:
let's change the behavior of `ossify` as well ⇒ to not revert if someone spoofed the storage, but unconditionally override

Reply:
probably can add ossify to the scope of proxy?

Reply:
to sum up, two classes of attacks:
- the issue described (ability to mint stETH against 'unknown' impl)
- the issue about 'honeypot' for new deployments: can't even connect to vaulthub if impl storage already spoofed
- DAO can't dismantle 'ossify' from new impls

Reply:
Looks like the 3rd one requires 'vault ownership' to be checked in the proxy, which is too heavy of a change to get in the last minute. See #1408 for an example.